### DISCUSSION TITLE
PixelProjection - (0, 0) at "Top Left" not working

### DISCUSSION THREAD

**Discussion by KOLEKTYW-online** [cables-gl/cables_docs#577] [! Welcome - readme first!]
In PixelProjection node Top Left center position doesn't work due to a typo in line 96. Space after "Left".
    if (inAlign.get() === "Top Left ")
should be
    if (inAlign.get() === "Top Left")

**Comment by steam0r** [comment]
wow, nice eye! fixed on dev.cables.gl and will be in todays release. thanks!

**Reply by steam0r** [reply]
done :)
